### PR TITLE
Add "Six Reasons I Chose Gatsby" blog post

### PR DESCRIPTION
**Pull request by raygesualdo**
Hey all. After this [conversation on Twitter](https://twitter.com/RayGesualdo/status/994562629040590848), I added my blog post. Let me know if I need to change anything. Thank you for all your hard work on Gatsby!

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Hey! Thanks for posting this! Excited for it to go live!

The only changes it seems are needed is a) remove the mention of your blog being open source since it doesn't make much sense here and b) make all the gatsbyjs.org links relative.

Thanks!

**Comment by pieh** [comment]
Preview link: https://deploy-preview-5369--gatsbyjs.netlify.com/blog/2018-05-11-six-reasons-i-chose-gatsby/

File: docs/blog/2018-05-11-six-reasons-i-chose-gatsby/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: Six Reasons I Chose Gatsby
+date: 2018-05-11
+author: Ray Gesualdo
+excerpt: "Spoiler alert: I'm a big fan of Gatsby."
+tags: ["gatsby", "react", "graphql", "plugins"]
+canonicalLink: https://www.raygesualdo.com/posts/six-reasons-i-chose-gatsby/
+publishedAt: raygesualdo.com
+---
+
+Spoiler alert: I'm a big fan of [Gatsby](https://gatsbyjs.org/). I've worked with it multiple times and I'm continually impressed with its power and flexibility. For those who aren't familiar, Gatsby is an open-source static site generator incorporating React and GraphQL. A few weeks ago, I switched this site to Gatsby and wanted to share my reasons for doing so. Plenty of articles have been written about _how_ to build a Gatsby site, but I wanted to talk about _why_ Gatsby is a great choice. I've outlined these reasons below in no particular order (the numbering is only for organizational purposes). I hope they give you a better understanding of Gatsby's benefits and features.
```
Comment (pieh) [inline docs/blog/2018-05-11-six-reasons-i-chose-gatsby/index.md]: > I switched this site to Gatsby

"this" refers to Your site not gatsbyjs.org, so need to change that

**Comment by jlengstorf** [comment]
Hiya @raygesualdo! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**